Dynatrace/dynatrace-operator
# Add support for oc debug pod command

# Description

Currently `oc debug pod` was not possible, as OpenShift removed the injected annotation from the debug pod and our webhook injected a second time. This was fixed by ignoring pods that have special OpenShift debug annotations.

## How can this be tested?

- Monitor with cloudNative or applicationOnly
- Run `oc debug <pod>` and check if it works

## Checklist
